modules/notify/main/index.coffee

```diff
@@ -1,33 +1,32 @@
 # utility method for setting up notifications
 setupNotification = (notification, selection) ->
-  if notification.options.icon? and notification.options.icon.length > 0
-    selection.append('div')
-      .class('hx-notification-icon-container')
-      .append('i')
-        .class('hx-notification-icon ' + notification.options.icon)
+  icon = if notification.options.icon? and notification.options.icon.length > 0
+    hx.detached('div').class('hx-notification-icon-container')
+      .add(hx.detached('i').class('hx-notification-icon ' + notification.options.icon))
 
-  selection
-    .append('div')
-      .class('hx-notification-text')
-      .text(notification.message)
+  content = hx.detached('div').class('hx-notification-content')
+  notification.options.renderer(content.node(), notification.message)
```

Comment:
We always run the renderer here now. 
This assumes the renderer hasn't been set explicitly to undefined - not sure if we need to add handling for that?